Change wording for List Voice Regions (#3649)
hammerandchisel/discord-api-docs: docs/resources/Voice.md

## List Voice Regions % GET /voice/regions

> Returns an array of [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) objects that can be used when setting the `rtc_region` on voice and stage channels.

Reviewer: ```suggestion
Returns an array of [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) objects that can be used when setting a voice or stage channel's [`rtc_region`](#DOCS_RESOURCES_CHANNEL/channel-object-channel-structure).
```
